app/styles/pages/_catalog.scss

```diff
@@ -37,12 +37,16 @@ $lines-to-show: 4;
   }
 }
 
+$container-height: 270px;
```

Comment:
Because the information was not the same as on the catalog page they felt really weird being as tall as the items on the catalog page.